src/excercise7.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Created by ttn on 28/12/16.
+ */
+/**
+ * Create a User Registration form with fields ( FirstName, LastName, EmailId, Contact Number, Password and Confirm password).
+ * Form should have a submit button. Apply form validation to incorporate following rules.
+ *
+ */
+
+
+import React from 'react'
+
+export default class UserRegisration extends React.Component {
+
+  constructor(props){
+    super(props)
+    this.state = {
+      firstNameTxt : "",
+      lastNameTxt : "",
+      emailTxt : "",
+      contactNumberTxt : "",
+      passwdTxt : ""
+    }
+
+    this.submitForm = this.submitForm.bind(this)
+
+  }
+
+
+  validateEmail(value){
+
+    let re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    return re.test(value);
+  }
+
+
+  submitForm(){
+
+    if(this.refs.firstNameRef.value > 0 && this.refs.lastNameRef.value > 0 ){
+      if(this.refs.firstNameRef.value === this.refs.lastNameRef.value){
```

Comment:
avoid using refs here.
Could have been done using onChange event handler and maintaining input value in state